Docs 8010 edit api groups pa (#255)
mulesoft/docs-api-manager: modules/ROOT/pages/api-groups-sla-tiers-faqs.adoc

= SLA Tiers For API Groups FAQs
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

*Question: What happens if an API Group is published without any SLA tiers?*

> *Answer:*

Reviewer: GLOBALLY REFORMAT per my Edit Tip last week on FAQS. See Style Reference.
I am sad that you missed that Slack message.

Author: Nope, this was meant to be removed so did not spend time on it.

> As a Group Administrator, you must carefully decide whether an API Group must not have any SLA tiers. No SLA implies unlimited SLA Tiers for all the instances in the APi group that do not have an SLA based policy applied. 

Reviewer: This text was not written by you. Please, you MUST edit Dev. input and you MUST reread your output before submitting to me.
```suggestion
As a Group Administrator, you must carefully consider whether an API Group is not allowed any SLA tiers. No SLA implies unlimited SLA tiers for all the instances in the API group that do not have an SLA-based policy applied. 
```

Author: Page to be removed. Done! Thanks!

> API Group instances with an SLA based policy will not be available for consumption until they have at least one tier defined. Group Administrators must decide to create SLA tiers after creating API Groups and before publishing the API Group instances to Exchange. 

Reviewer: ```suggestion
API Group instances with an SLA-based policy are not available for consumption until they have at least one tier defined. Group Administrators must decide to create SLA tiers after creating API Groups and before publishing the API Group instances to Exchange. 
```

Author: Page to be removed. Done! Thanks!

*Question: Can you modify API Groups SLA tiers if there is an active contract?* 
*Answer:*

> Yes you can update the limits. It works the same today for API level SLA tiers.

Reviewer: today?
API-level

Author: Page to be removed. Done! Thanks!

> *Question: When can I delete/deprecate a tier?*

Reviewer: delete or deprecate

Author: Page to be removed. Done! Thanks!

*Answer:*

> If a contract is in use, it cannot be deleted, but only deprecated.  Deprecated tiers maintain the previous contracts but do not allow any new contracts. 

Reviewer: it > the tier

Author: Page to be removed. Done! Thanks!

*Question: Do I need to maintain two different tiers: one on API instance and another on Group level?*

*Answer:*
API Owners can decide if an API can be accessed directly or only through an API Group.
We can add a new state for APIs, where we can mark the API as direct access restricted, and with that restrict the Request API Access. The condition to show the button would now be that the API isn’t deprecated or restricted. 

> That would cover the case where API owners implement APIs that are to belong to a API Groups and are not interested in maintaining the tiers of it.

Reviewer: This needs rewriting. We cannot publish text straight from Dev.

Author: Page to be removed. Done! Thanks!
